.github/copilot-instructions.md

```diff
@@ -46,9 +54,10 @@ This is a Python based repository, it provides a Django reusable application tha
 - **User creation**: Use the `make_user()` utility function in combination with the `user_factory` attribute
   - **Admin users**: Set the `user_factory` to `SuperUserFactory`
 - **Authentication**: Use `self.login(self.user)` approach 
-- **HTTP requests**: Use `self.get(...)` instead of direct client calls
+- **HTTP requests**: Use `self.get(...)` instead of direct client calls with `self.reverse()` for named URLs
```

Comment:
This wasn't what I wanted, the get already expects the named url rather than it already being reversed. Reverse this suggestion and update the tests to use `self.get(named_url, *args)` style.